templates/dashboard/support_disabled.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'default.html' %}
+
+{% set active_page = None %}
+
+{% block title %}
+    Support
+{% endblock %}
+
+{% block head %}
+    <style>
+        .card-title {
+            font-size: 22px;
+            font-weight: 600;
+            margin-bottom: 3px;
+        }
+    </style>
+{% endblock %}
+
+{% block default_content %}
+
+    <div class="col pb-3">
+        <!-- Current plan -->
+        <div class="card">
+            <div class="card-body">
+                <div class="card-title mb-3">Support form is disabled. Please configure Zendesk integration.</div>
```

Comment:
Could we instead use an error through `flash("Support form is disabled. Please configure Zendesk integration.", "error")` through the backend and then redirect the user back to the dashboard rather than having a dedicated page for it?

Example:
![image](https://user-images.githubusercontent.com/23139726/153260316-c9b2c9b9-b531-463a-a3ff-414076ffaee9.png)

Although the "error" modal will be red instead of orange.